FAQ.md

```diff
@@ -189,16 +187,15 @@ Due to a bug in Android this is not possible.
 See [here](https://github.com/M66B/NetGuard/issues/42) for more information.
 
 <a name="FAQ24"></a>
-**(24) Can you remove the notification from the status bar?**
+**(24) Can you remove the notification from the notification screen?**
```

Comment:
Okay, I see, but to understand this correctly, you need to know that the status bar notification icon is shown in the status bar even when it is low priority if there are not many other icons to show. The exact behavior is not known to me, since it is not documented. So the notification can and will be visible in both the status bar and the pull down notification screen, depending on the circumstances.